build/raspberry-pi/init_resize.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/sh
+
+reboot_pi () {
```

Comment:
:speech_balloon: Didn't realize that you didn't need function at the start.
Looked up and this is correct, just commenting to promote active reading.